ca/server.go

```diff
@@ -69,6 +69,26 @@ func (s *Server) SetReconciliationRetryInterval(reconciliationRetryInterval time
 	s.reconciliationRetryInterval = reconciliationRetryInterval
 }
 
+// GetUnlockKey is responsible for returning the current unlock key used for encrypting TLS private keys and
+// other at rest data.  Access to this RPC call should only be allowed via mutual TLS from managers.
+func (s *Server) GetUnlockKey(ctx context.Context, request *api.GetUnlockKeyRequest) (*api.GetUnlockKeyResponse, error) {
+	log.G(ctx).WithFields(logrus.Fields{
+		"method": "GetUnlockKey",
+	})
+
+	var managerUnlockKey []byte
+	s.store.View(func(tx store.ReadTx) {
+		cluster := store.GetCluster(tx, s.securityConfig.ClientTLSCreds.Organization())
+		if cluster.Spec.EncryptionConfig.AutoLockManagers {
+			managerUnlockKey = cluster.UnlockKeys.Manager
+		}
+	})
+
+	return &api.GetUnlockKeyResponse{
+		UnlockKey: managerUnlockKey,
```

Comment:
While I don't disagree that at some point, we may want to differentiate different types of managers, this conversation seems to tie in a bit more with moving all the encryption to above raft, and possibly having different symmetric keys for different types of data.

I'm not sure it's worth actually adding the asymmetric encryption in at this point, but perhaps we can wrap the unlock key can be a `MaybeEncryptedRecord`?  That way we don't have to add a new field at any point?